readme.md

```diff
@@ -228,72 +331,75 @@ You might have noticed when we ran `rspec` before that there was a little asteri
-
-## Formatting the output
-
-Pass the "format" flag to the rspec command:
+---
 
 ```
-rspec --color --format documentation
-```  
+..
 
-The default format is "progress", it's designed to show the result of many tests, concisely.  Until we have a bunch of tests, I prefer the "documentation" format.  It helps me to ask the right questions.  You can run it with the shorthand, `rspec -c -fd`.
+Finished in 0.0005 seconds (files took 0.07273 seconds to load)
+2 examples, 0 failures
 
-If you prefer this as your default, you can add it to RSpec's configuration file, `~\.rspec`.  Here's mine:
-
-```
---color
---format documentation
 ```
 
+Adding `do...end` makes RSpec think this test is an actual test -- not pending anymore. There's no malfunctioning code inside this test, so RSpec is saying it passes. Asterisk `*` indicates a pending test, and dot `.` indicates a passing test.
+
+## Break (10 min)
 
-## Actually testing something
+## We-Do: Passing Our First Test (10 min)
 
-Let's make these tests actually test something. Inside the first test, make (but don't save) a new Apartment and save it to a variable.
+Let's make these tests actually test something. Inside the first test, make (but don't save) a new Dog and save it to a variable.
 
 ```rb
-it "has the class Apartment" do
-  apartment = Apartment.new
+it "has the class Dog" do
+  dog = Dog.new("Rover", 10)
 end
 ```
 
-Now, let's tell the test that we expect that new apartment to be an apartment:
+Now, let's tell the test that we expect that new dog to be a dog:
 
 ```rb
-it "has the class Apartment" do
-  apartment = Apartment.new
-  expect(apartment).to be_a(Apartment)
+it "has the class Dog" do
+  dog = Dog.new("Rover", 10)
+  expect(dog).to be_a(Dog)
 end
 ```
 
 Run `rspec`. The test should still pass.
 
-Read that new line to yourself. We literally wrote "expect apartment to be a apartment". This is totally proper English, except for us using "a" instead of "an"!
+>Read that new line to yourself. We literally wrote "expect dog to be a dog". This is totally proper English but it's code! The methods that come built-in with RSpec were created in such a way that tests look as English-y as possible.
 
-...but it's code! The methods that come built-in with RSpec were created in such a way that tests look as English-y as possible.
+This is our first first spec or "expectation"
 
-Now let's make the test **fail**. Let's say we "expect apartment to be a Integer":
+RSpec assertions have two components: `expectation` and `matcher`.
+
+1. We expect `dog` to be something.
+
+2. We identify that "something" with a Matcher:
+
+> Expectation: `expect(dog).to `
+
+> Matcher: `be_a(Dog)`
+
+>We use `expect(IUT)` to "wrap" the ***Item Under Test***, so that it supports the `to` method which accepts a matcher. Here we are wrapping an object or block in expect, call to or to_not (aliased as not_to) and pass it a matcher object
+
+Now let's make the test **fail**. Let's say we "expect dog to be a Integer":
```

Comment:
imagine people are gonna ask about other possible matchers here - include a link to documentation